APCAppCore/APCAppCore/Startup/APCTabBarViewController.m

```diff
@@ -67,8 +66,6 @@ - (void)passcodeViewControllerDidSucceed:(APCPasscodeViewController *)viewContro
     self.isPasscodeShowing = NO;
     self.showPasscodeScreen = NO;
     [viewController dismissViewControllerAnimated:YES completion:nil];
-    [[NSUserDefaults standardUserDefaults] setObject: [NSDate date] forKey:kLastUsedTimeKey];
```

Comment:
These 2 lines can be added to AppDelegate's `passcodeViewControllerDidSucceed` implementation.